src/@dvcorg/gatsby-theme-iterative/components/LayoutHeader/alert.tsx

```diff
@@ -11,15 +10,15 @@ const LayoutAlert: React.FC<{ collapsed: boolean }> | false = ({
 }) => (
   <div className={cn(styles.alert, collapsed && styles.collapsed)}>
     <div className={styles.text}>
-      <span role="img" aria-label="VS Code Logo">
-        <VscodeIcon width="22px" height="22px" />
-      </span>
+      <span role="img" aria-label="rocket">
+        🚀
+      </span>{' '}
       <p>
-        Check out our{' '}
-        <Link href="https://marketplace.visualstudio.com/items?itemName=Iterative.dvc">
-          new VS Code extension
+        New Release!{' '}
+        <Link href="https://dvc.org/blog/iterative-studio-model-registry">
+          Git-backed Machine Learning Model Registry
```

Comment:
The link checker can't find this one but it's OK because it's in a [different PR](https://github.com/iterative/dvc.org/pull/3800). Review app:

https://dvc-org-mr-blog-post-ztcaladh1.herokuapp.com/blog/iterative-studio-model-registry